java/src/org/openqa/selenium/grid/router/GridStatusHandler.java

```diff
@@ -141,6 +141,7 @@ public HttpResponse execute(HttpRequest req) {
                           .put("id", node.getNodeId())
                           .put("uri", node.getExternalUri())
                           .put("maxSessions", node.getMaxSessionCount())
+                          .put("sessionTimeout", node.getSessionTimeout().toMillis())
                           .put("osInfo", node.getOsInfo())
                           .put("heartbeatPeriod", node.getHeartbeatPeriod().toMillis())
                           .put("availability", node.getAvailability())
```

Comment:
Why not calling the `node.toJson()` method here?
(The visibility of the method has to be changed to public.)

Reply:
Yes, make sense. I updated it, will prevent mismatch if Node has new attributes later.